connect/api.go

```diff
@@ -84,3 +84,41 @@ func deactivateProduct(product Product) (Service, error) {
 	err = json.Unmarshal(resp, &remoteService)
 	return remoteService, err
 }
+
+// updateSystem updates the system's hardware info on SCC
+// https://scc.suse.com/connect/v4/documentation#/systems/put_systems
```

Comment:
please add a similar comment about the format of this data and `makeSysInfoBody()`

Reply:
done